x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/layout/policy_event_filters_layout.test.tsx

```diff
@@ -12,24 +12,22 @@ import {
   createAppRootMockRenderer,
 } from '../../../../../../common/mock/endpoint';
 import { EndpointDocGenerator } from '../../../../../../../common/endpoint/generate_data';
-import { EventFilterGenerator } from '../../../../../../../common/endpoint/data_generators/event_filter_generator';
 
-import { EventFiltersHttpService } from '../../../../event_filters/service';
 import { ImmutableObject, PolicyData } from '../../../../../../../common/endpoint/types';
 import { parsePoliciesAndFilterToKql } from '../../../../../common/utils';
-
-jest.mock('../../../../event_filters/service');
-const EventFiltersHttpServiceMock = EventFiltersHttpService as jest.Mock;
+import { eventFiltersListQueryHttpMock } from '../../../../event_filters/test_utils';
```

Comment:
I updated this test file to use eventFiltersListQueryHttpMock instead of mocking the service manually